<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindowUi</class>
 <widget class="QWidget" name="GraphWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H2GraphWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
     </property>
     <property name="cacheMode">
      <set>QGraphicsView::CacheNone</set>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::MinimalViewportUpdate</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonExportPNG">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Graphs.qrc">
         <normaloff>:/images/PNG_72x72.png</normaloff>:/images/PNG_72x72.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>72</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Graphs.qrc">
         <normaloff>:/images/SVG_72x72.png</normaloff>:/images/SVG_72x72.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>72</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Display Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Threshold:</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sbThreshold">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonTimeCourses">
              <property name="toolTip">
               <string>Visualise the evolution of values in time for a pair of channels</string>
              </property>
              <property name="text">
               <string>Plot Values In Time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonMean">
              <property name="text">
               <string>Show Mean Graph</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="checkH2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Show Values</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkHideSensors">
              <property name="text">
               <string>Hide Non Connected Sensors</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkAsym">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Show Asymetry</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Zoom:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>60</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>80</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>120</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>150</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Navigation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Position in file:</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>spinPosition</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinPosition">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <double>9999999999999999635896294965248.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Section:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="buttonBack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonNext">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonAnimate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Graphs.qrc">
              <normaloff>:/images/Play-icon_32x32.png</normaloff>:/images/Play-icon_32x32.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Delay (s):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbDelay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbLoop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Loop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
  <include location="Graphs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonExportPNG</sender>
   <signal>clicked()</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>exportGraphToPNG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>677</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>exportGraphToSVG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>previousTimeWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>885</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>721</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>nextTimeWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>891</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>746</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAnimate</sender>
   <signal>clicked()</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>animate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>895</y>
    </hint>
    <hint type="destinationlabel">
     <x>990</x>
     <y>707</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbLoop</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>setLoop(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>895</y>
    </hint>
    <hint type="destinationlabel">
     <x>990</x>
     <y>641</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkH2</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>showValues(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>827</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAsym</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>showAsymetry(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>990</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboSize</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>changeViewSize(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>990</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkHideSensors</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWindowUi</receiver>
   <slot>hideSensors(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>820</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>871</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exportGraphToPNG()</slot>
  <slot>exportGraphToSVG()</slot>
  <slot>changeThreshold(int)</slot>
  <slot>previousTimeWindow()</slot>
  <slot>nextTimeWindow()</slot>
  <slot>animate()</slot>
  <slot>setLoop(bool)</slot>
  <slot>showValues(bool)</slot>
  <slot>showAsymetry(bool)</slot>
  <slot>changeViewSize(QString)</slot>
  <slot>showH2Signals()</slot>
  <slot>hideSensors(bool)</slot>
 </slots>
</ui>
